CONTRIBUTING.md

```diff
@@ -7,20 +7,18 @@ and agree by submitting the patch
 that your contributions are licensed under
```

Comment:
These changes look great! Should we also add a note somewhere that guidance/tips for writing great documentation for TSPL is forthcoming and link to a GitHub issue that tracks it? It would publicise this effort the documentation workgroup is pursuing, and could invite contributors to participate.